wp-content/themes/newspack-theme/acf-json/index.php

```diff
@@ -0,0 +1,3 @@
+<?php
+// Silence is golden.
+?>
```

Comment:
This file is recommended in the acf-json settings page. Keeps the files private.